test/features/steps/utils.py

```diff
@@ -46,7 +46,24 @@ def add_sudo(cmd):
     return cmd
 
 
+def _wrap_cmd_non_root(cmd):
```

Comment:
Comment is needed here to document what this function serves.

Reply:
Added